web/ui/dashboard/src/app/private/components/create-portal-link/create-portal-link.component.html

```diff
@@ -1,91 +1,81 @@
-<div convoy-modal position="full" (closeModal)="goBack()">
-	<div convoy-modal-header fullscreen="true">
-		<div class="flex items-center">
-			<button convoy-button size="sm" texture="light" class="px-10px !py-10px" (click)="goBack()">
-				<img src="/assets/img/modal-close-icon.svg" class="w-12px h-12px" alt="close icon" />
-			</button>
+<div convoy-modal-header fullscreen="true">
+	<div class="flex items-center">
+		<button convoy-button size="sm" texture="light" class="px-10px !py-10px" (click)="goBack()">
+			<img src="/assets/img/modal-close-icon.svg" class="w-12px h-12px" alt="close icon" />
+		</button>
```

Comment:
Is there a way to make this back button come by default with this modal header directive?